boards/arm/pan1781_evb/pan1781_evb.dts

```diff
@@ -0,0 +1,150 @@
+/*
+ * PAN1781 EVB configuration
+ *
+ * Copyright (c) 2022 Panasonic Industrial Devices Europe GmbH
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52820_qdaa.dtsi>
+#include "pan1781_evb-pinctrl.dtsi"
+
+/ {
+	model = "pan1781-evb";
+	compatible = "panasonic-industrial-devices-europe-gmbh,pan1781-evb";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		evb_led1: evb_led_0 {
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+			label = "LED1 on EVB";
+		};
+		evb_led2: evb_led_2 {
+			gpios = <&gpio0 15 GPIO_ACTIVE_LOW>;
+			label = "LED2 on EVB";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		pwm_evb_led1: pwm_evb_led_1 {
+			pwms = <&sw_pwm 0 PWM_MSEC(20) PWM_POLARITY_INVERTED>;
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		evb_sw1: evb_sw_1 {
+			gpios = <&gpio0 4 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "SW1 on EVB";
+		};
+		evb_sw2: evb_sw_2 {
+			gpios = <&gpio0 5 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "SW2 on EVB";
+		};
+	};
+
+	arduino_header: connector {
+		compatible = "arduino-header-r3";
+		#gpio-cells = <2>;
+		gpio-map-mask = <0xffffffff 0xffffffc0>;
+		gpio-map-pass-thru = <0 0x3f>;
+		gpio-map = <0 0 &gpio0 2 0>,   /* A0 */
+				   <1 0 &gpio0 3 0>,   /* A1 */
+				   <2 0 &gpio0 4 0>,   /* A2 */
+				   <3 0 &gpio0 5 0>,   /* A3 */
+				   <6 0 &gpio0 6 0>,   /* D0 */
+				   <7 0 &gpio0 8 0>,   /* D1 */
+				   <8 0 &gpio0 14 0>,  /* D2 */
+				   <9 0 &gpio0 15 0>,  /* D3 */
+				   <10 0 &gpio0 4 0>,  /* D4 */
+				   <11 0 &gpio0 5 0>,  /* D5 */
+				   <16 0 &gpio0 3 0>,  /* D10 */
+				   <17 0 &gpio0 20 0>, /* D11 */
+				   <18 0 &gpio0 29 0>, /* D12 */
+				   <19 0 &gpio0 28 0>, /* D13 */
+				   <20 0 &gpio0 7 0>,  /* D14 */
+				   <21 0 &gpio0 30 0>; /* D15 */
+	};
```

Comment:
if it has some entries missing, maybe it should not be considered an `arduino-header-r3` then? cc: @anangl

Reply:
Hmm okay. Yeah maybe you are right. When the `arduino-header-r3 ` is defined like it is in "arduino-header-r3.yaml" everyone (including the twister tests) assumes, that the chip is really connected to all of the A0-A5 and the D0-D15 pins.
On the other hand, it's really unpleasant that we have to remove the whole header now, because just some of them are missing. But if iam got you right, there is no other option to succeed the twister runs and finally complete the upload of our boards? So if there is no arduino header present in the device tree, the shield tests will not be active for the twister runs on our board?

Reply:
I guess one solution is to create a compatible specific to your connector variant, and update the `samples/subsys/display/lvgl/sample.yaml` so that it depends on connector r3 compatible for the `sample.display.lvgl.adafruit_2_8_tft_touch_v2` test.

that is, add:

```
filter: dt_compat_enabled("arduino-connector-r3")
```

Reply:
FYI, I have started fixing some tests: https://github.com/zephyrproject-rtos/zephyr/pull/46835. Note that you should create a new compatible for your connector, I see no other way to reliably exclude tests that expect a complete arduino R3 connector. Adding this board to the exclude list because it misses one pin is not maintainable, as it will continue impacting new additions to the tree. For now, the compatible won't do anything until tests are improved, so your exclude list needs to be kept.

Reply:
Hi @gmarull 
Now I have created the compatible `panasonic,reduced-arduino-header` and changed the compatible for the pan1782_evb and pan1782_evb in the dts files. These used the arduino-header-r3 before. Hope that fits for you now?